packages/nextjs/app/builders/0x6BF7d6b94282BD48ff458599aDafA268BcB009FF/page.tsx

```diff
@@ -0,0 +1,121 @@
+import { IconType } from "react-icons";
+import { FaEnvelope, FaGithub, FaLinkedin, FaMapMarkerAlt, FaTwitter } from "react-icons/fa";
+
+interface ProfileData {
+  name: string;
+  title: string;
+  bio: string;
+  avatar: string;
+  location: string;
+  email: string;
+  social: {
+    github: string;
+    twitter: string;
+    linkedin: string;
+  };
+}
+
+interface StyledIconProps extends React.HTMLAttributes<HTMLSpanElement> {
+  icon: IconType;
+  size?: number;
+  color?: string;
+}
+
+const StyledIcon: React.FC<StyledIconProps> = ({ icon: Icon, className, size, color, ...props }) => {
+  return (
+    <span className={`inline-block ${className}`} {...props}>
+      <Icon size={size} color={color} />
+    </span>
+  );
+};
+
+const ProfileCard: React.FC = () => {
+  const profile: ProfileData = {
+    name: "Similoluwa Abidoye",
+    title: "Full Stack Developer",
+    bio: "Passionate about building the future of Web3 with modern technologies. Love working with React, TypeScript, and Solidity, hardhat and soon. Always learning, always coding.",
+    avatar: "/simze.jpg",
```

Comment:
Where is the image file? I cannot find it. 

<img width="447" alt="Screenshot 2024-11-20 at 15 24 58" src="https://github.com/user-attachments/assets/ffb8fcec-7c79-432d-8a9c-ba4e0d748ac5">